src/classlibnative/bcltype/system.cpp

```diff
@@ -225,17 +225,18 @@ FCIMPL1(Object*, SystemNative::_GetEnvironmentVariable, StringObject* strVarUNSA
     int len, newLen;
 
     // Get the length of the environment variable.
-    WCHAR dummy;    // prefix complains if pass a null ptr in, so rely on the final length parm instead
-    len = WszGetEnvironmentVariable(strVar->GetBuffer(), &dummy, 0);
+    PathString dummy;    // prefix complains if pass a null ptr in, so rely on the final length parm instead
```

Comment:
This variable shouldn't be called dummy any more, since it will contain the actual value after this change.